lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +22,96 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  color: $nx-text-color-light;
+  display: grid;
+  grid-template-columns: 85% auto;
+  grid-template-rows: auto auto;
   margin: 0;
-  padding: $nx-spacing-sm $nx-spacing-xs;
+  padding: $nx-spacing-md;
 
   // we use first of type because we only want to modify the first list item,
   // not alerts, errors, or empty states
   &:first-of-type {
-    border-top: $nx-border;
+    border-top: 1px solid #d1d6f0;
   }
+}
 
-  &.nx-error {
-    border: $nx-border-error;
-    color: $nx-text-color-error;
-  }
+.nx-list__text {
+  @include semi-bold();
 
-  > .fa, > .svg-inline--fa {
-    margin-right: $nx-spacing-md;
-  }
+  grid-column: 1;
+  grid-row: 1;
 }
 
-.nx-list--bulleted {
-  .nx-list__item {
-    @include regular();
+.nx-list__subtext {
+  @include regular();
 
-    border: none;
-    list-style: disc outside;
-    margin: $nx-spacing-sm 10px $nx-spacing-sm 36px;
-    overflow: visible;
-    padding: 0;
-    white-space: normal;
-  }
+  align-self: start;
+  display: block;
+  grid-column: 1;
+  grid-row: 2;
+  // top margin is measured down from the top of the objects grid row
+  margin: $nx-spacing-sm 0 0 0;
+  white-space: normal;
 
-  .nx-list__item {
-    list-style-type: disc;
+  &.nx-truncate-ellipsis {
+    white-space: nowrap;
+    width: $nx-paragraph-width-maximum;
```

Comment:
I think this needs to be moved out of this block.  The wrapping variant does not wrap at the paragraph width, which I think it's supposed to.

Reply:
> I think this needs to be moved out of this block. The wrapping variant does not wrap at the paragraph width, which I think it's supposed to.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/68446a6e5fd7846bda4b38d58f16434b4d1b6ac8

Reply:
> > I think this needs to be moved out of this block. The wrapping variant does not wrap at the paragraph width, which I think it's supposed to.
>
> 68446a6

Now they go all the way across before wrapping.  I thought the paragraph width was still desired when they wrap.

![image](https://user-images.githubusercontent.com/3630091/93785243-db21a600-fbfb-11ea-81c2-bec8a4c24a89.png)

Reply:
> > > I think this needs to be moved out of this block. The wrapping variant does not wrap at the paragraph width, which I think it's supposed to.
> > 
> > 
> > [68446a6](https://github.com/sonatype/sonatype-react-shared-components/commit/68446a6e5fd7846bda4b38d58f16434b4d1b6ac8)
> 
> Now they go all the way across before wrapping. I thought the paragraph width was still desired when they wrap.
> 
> ![image](https://user-images.githubusercontent.com/3630091/93785243-db21a600-fbfb-11ea-81c2-bec8a4c24a89.png)

The Zeplin file shows the sub-text as full width so I'd assume that's when truncation occurs as well.

Reply:
> > > > I think this needs to be moved out of this block. The wrapping variant does not wrap at the paragraph width, which I think it's supposed to.
> > > 
> > > 
> > > [68446a6](https://github.com/sonatype/sonatype-react-shared-components/commit/68446a6e5fd7846bda4b38d58f16434b4d1b6ac8)
> > 
> > 
> > Now they go all the way across before wrapping. I thought the paragraph width was still desired when they wrap.
> > ![image](https://user-images.githubusercontent.com/3630091/93785243-db21a600-fbfb-11ea-81c2-bec8a4c24a89.png)
> 
> The Zeplin file shows the sub-text as full width so I'd assume that's when truncation occurs as well.

I did trim some unnecessary styling here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/c9272ec404c887a80481b5c82a28de928129256d